server/controllers/email-report/index.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * @module email-report/email-report
+ *
+ * @description
+ * small description of this module
+ *
+ * @requires db
+ * @requires node-uuid
+ * @requires BadRequest
+ * @requires NotFound
+ */
+
+
+const uuid = require('node-uuid');
+
+const db = require('../../lib/db');
+const Topic = require('../../lib/topic');
+
+const BadRequest = require('../../lib/errors/BadRequest');
+const NotFound = require('../../lib/errors/NotFound');
+const Period = require('../../lib/period');
+
+exports.create = create;
+exports.list = list;
+exports.list_people = list_people;
```

Comment:
Please use `camelCase` instead of `underscore_case`.  This should be called `listPeople`.